templates/Register.html

```diff
@@ -23,22 +26,114 @@
         <input name="spotifytoken" id="spotify" type="checkbox">
     </p>
 
+
     <Label for="pin">4 character pin: </Label>
     <input name="pin" type="text" Placeholder="****" required pattern=".{4,4}" maxlength="4" minlength="4">
     <br>
     <p>Picture of face:</p>
-    <input name="facepath"  type="file" accept="image/*" capture="camera" required>
+    <input name="facepath" id="facepath"  type="file" accept="image/*" capture="camera" required>
 
     <hr/>
-
-    <input class="btn btn-lg btn-success" type="submit">
+    <!--can't get to post the way I want currently
+    "time();calendar(); spotifytoken(); googletoken(); maps(); pin(); weather();twittertoken(); name();email(); facepath(); postIt(); printC()"-->
+    <input onclick= "test()" class="btn btn-lg btn-success" type="submit">
     <input class ="btn btn-lg btn-danger" type="button" onclick="window.location.href='{{url_for('index')}}'" value="Cancel" />
 </form>
 
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
+<script>
+    var userChoices={
+        name:"name" ,
+        email:"rando@gmail.com",
+        googletoken:"temp",
+        pin:"temp",
+        spotifytoken: false,
+        twittertoken: false,
+        maps: false,
+        calendar:false,
+        weather:true,
+        time:true
+    }
+    //tested time() and printed userchoices.time
+function test(){
+        if (document.getElementById('time').checked)
+            alert("shit hit the fan");
+        else {
+            facePath();
+            alert(userChoices.facepath);
+        }
+}
+function time(){
+    if(document.getElementById('time').checked) {
+        userChoices.time=true;
+    } else {
+        userChoices.time=false;
+}
+}
+function calendar(){
+    if(document.getElementById('calendar').checked) {
+        userChoices.calendar=true;
+    } else {
+        userChoices.calendar=false;
+}
+}
+function spotifytoken(){
```

Comment:
remember these are flags now not tokens. You can check the naming convention on the DB to match this.

Reply:
Yea, they weren't changed at the time of creation so I haven't changed them yet.